Makefile

```diff
@@ -204,7 +204,7 @@ local-dev-up: local-dev-clean node_modules | build-image-quay
 	DOCKER_USER="$$(id -u):$$(id -g)" $(DOCKER_COMPOSE) up -d --force-recreate local-dev-frontend
 	$(DOCKER_COMPOSE) up -d redis quay-db
 	$(DOCKER) exec -it quay-db bash -c 'while ! pg_isready; do echo "waiting for postgres"; sleep 2; done'
-	DOCKER_USER="$$(id -u):0" $(DOCKER_COMPOSE) up -d quay
+	DOCKER_USER="$$(id -u):0" QUAY_FEATURE_SECURITY_SCANNER=false $(DOCKER_COMPOSE) up -d quay
```

Comment:
isni't this controlled by `local-dev/stack/config.yaml` don't think we need to disable via env variable

Reply:
We have `local-dev-up` and `local-dev-up-with-clair`, so far they both use the same config, but the security scanner worker cannot work without Clair and generates noise in logs. Either we need a new config file, or disabling features via environment variables.